README.md

```diff
@@ -1,46 +1,44 @@
-This [Composer](https://getcomposer.org) project template is a quick way to install
-the [Apigee Developer Portal Kickstart installation profile](https://www.drupal.org/project/apigee_devportal_kickstart)
-for Drupal.
+# Apigee Developer Portal Kickstart Composer Project
+
+This [Composer](https://getcomposer.org) project template is a quick way to install the [Apigee Developer Portal Kickstart installation profile](https://www.drupal.org/project/apigee_devportal_kickstart) for Drupal.
 
 This is a Composer package that installs the following components to make a fully working and configured site:
+
 * [Drupal Core](https://www.drupal.org)
 * [Apigee Developer Portal Kickstart installation profile](https://www.drupal.org/project/apigee_devportal_kickstart)
 
-# Prerequisites
+## Prerequisites
 
 * [Git](https://git-scm.com)
 * [Composer](https://getcomposer.org)
 
-# Installation
+## Installation
 
-The following command will download Drupal core and the Apigee Developer Portal Kickstart profile into
-the MY_PROJECT directory:
+The following command will download Drupal core, the Apigee Developer Portal Kickstart profile, and their dependencies, into the `MY_PROJECT` directory:
 
-```
+```sh
 composer create-project apigee/devportal-kickstart-project:8.x-dev MY_PROJECT --stability dev --no-interaction
 ```
 
-The actual web root will be MY_PROJECT/web. You will need to point your web server to serve up that
-directory and run the installer like any Drupal site installation.
+The actual web root will be `MY_PROJECT/web`. You will need to point your [web server](https://www.drupal.org/docs/develop/local-server-setup) to serve that directory. Then, visit the site in a web browser, and you'll be redirected to `core/install.php`, where you can run the installer like any Drupal site installation. Running the installation via web browser is recommended for the best experience.
```

Comment:
Is there a doc that we can point people to instead on Drupal.org?  We can say "setup and install following standard Drupal install instructions at ?"

Reply:
@cnovak I linked to this ^ https://www.drupal.org/docs/develop/local-server-setup, which is a step in the installation guide on drupal.org.  We can shorten this section, and just point there, and keep the last sentence about using the browser, if you'd like.

There are a few guides on Drupal.org...

1. Evaluator guide (which uses composer quick-start):
    - https://www.drupal.org/docs/official_docs/en/_evaluator_guide.html 
2. User guide (most complete): 
    - https://www.drupal.org/docs/user_guide/en/index.html
    - Installation chapter: https://www.drupal.org/docs/user_guide/en/installation-chapter.html
3. Local development guide (which documents using DDEV):
    - https://www.drupal.org/docs/official_docs/en/_local_development_guide.html